### ISSUE TITLE
Caused by: java.lang.SecurityException: com.androidfu.example.geofences from uid 10049 not allowed to perform MOCK_LOCATION

### ISSUE THREAD

**Issue by Narender8** [androidfu/GeofenceExample#2]
showing me this error on resume at below line
        Log.i(TAG, "GPS Provider");
        locationManager.addTestProvider(LocationManager.GPS_PROVIDER, false, true, false, false, false, false, false, Criteria.POWER_HIGH, Criteria.ACCURACY_FINE);
        locationManager.setTestProviderEnabled(LocationManager.GPS_PROVIDER, true);

**Comment by qteen1** [comment]
Yeah i am getting that error as well, did you find a solution to it?

**Comment by herbert-soares-ifood** [comment]
Hello, did you find a solution to it?
